### PR TITLE
:sparkles: feat: Added LBCard into project

### PR DESCRIPTION
**Pull request by hartmannjean**
## Task

This pull request refers to task LOR-136

- [ ] Does not apply to this pull request

## Description

Added LBCard in the UI components module
Added colors for the created component
Added missing images
Added texts in the string.xml file

## Types of changes #

- [X] New feature (non-breaking change which adds functionality).
- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).

## What changed: #

- [X] UI
- [X] Business rule
- [ ] Documentation
- [ ] Gradle or Build
- [ ] Navigation
- [ ] Tests
- [X] Resource files
- [ ] Project architecture
- [ ] Code style

## Checklist #

- [X] Have tested the changes.
- [ ] Met all the acceptance requirements of this task.
- [X] Verified if branch is up-to-date with `development` (if not - rebase it or merge it).
- [X] Resolve git conflicts

## If any of the acceptance criteria are not met, please explain why below #
What is the criteria? Why it is different?

### CODE REVIEW

**Review by softdevandre — APPROVED** [review]
:ok_hand:

**Comment by softdevandre** [comment]
> 👌

After resolve the strings.xml conflict with develop branch it's all done, bro. Thank you for your work.

File: core/ui/src/main/java/com/loryblu/core/ui/components/LBCard.kt [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    idImage: Int,
+    image: Int,
+    text: Int,
+    modifier: Modifier,
+    onclick: () -> Unit,
+    isCardDisabled: Boolean = false
+) {
+    var cardClicked by rememberSaveable {
+        mutableStateOf(false)
+    }
```
Comment (OdisBy) [inline core/ui/src/main/java/com/loryblu/core/ui/components/LBCard.kt:62]: This will be passed as parameter.

**Review by OdisBy — APPROVED** [review]
💯 boa meu mano

**Pull request merged**